puppet/modules/web/manifests/vhost/web.pp

```diff
@@ -89,4 +89,20 @@
       script_content => file('web/rsync.sh'),
     }
   }
+
+  # Generate RSS stats
+  file { '/var/log/rss-stat':
+    ensure => directory,
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0750',
+  }
+
+  file { '/etc/cron.weekly/rss-stat':
```

Comment:
This is created via `/etc/logrotate.d/httpd`. I'm not sure I want to modify that, but I wonder if that would be better. I'm a bit worried that if we run this weekly and logrotate runs weekly that it may be a week behind most of the time (because cron would run before logrotate and then wait a week).

Still, we keep 4 weeks of backlog so it won't really be a big problem.

Reply:
When talking to @tbrisker, we both agreed that making this executed from `logrotate` directly would be cool, but none of us is well-versed in `logrotate` and I always found it rather awful to debug, so didn't want to touch.

And yes, this makes us probably lag by a week, but then again, right now we lag by 3 weeks until @tbrisker does it ;)